recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "v0.5.1" %}
+{% set version = "v0.5.2" %}
 
 package:
   name: nbgrader
   version: {{ version }}
 
 source:
   url: https://github.com/jupyter/nbgrader/archive/{{ version }}.tar.gz
```

Comment:
Don't worry. I will have to pull the other packages from the channel though. (I am only waiting for this PR.)